### DISCUSSION TITLE
[Bug] Manual sort doesn't work in v0.64.3

### DISCUSSION THREAD

**Discussion by sudoexec** [outline/outline#3600] [Self Hosting] [answered]
In the latest v0.64.3 release. Manul sort doesnt' work anymore.

**Comment by tommoor** [comment, marked as answer]
Hi, can't reproduce – please provide more details

**Reply by sudoexec** [reply]
It looks like the browser version causes this.
Chromium 102.0.5005.61 (Official Build) Arch Linux (64-bit) has this issue, but chrome and unstable chrome don't have this issue.
